### PR TITLE
Emscripten 3.1.20

### PR DESCRIPTION
**Pull request by hoodmane**
GSW patch:
https://github.com/TEOS-10/GSW-Python/pull/99

galpy patch:
https://github.com/jobovy/galpy/pull/499

pyerfa patch:
https://github.com/liberfa/pyerfa/pull/89

astropy patch:
https://github.com/astropy/astropy/pull/13586

scipy patch:
https://github.com/scipy/scipy/pull/16924

### CODE REVIEW

**Comment by hoodmane** [comment]
@ryanking13 could you review this? I think it's good to merge.

File: packages/pyerfa/meta.yaml [hunk 1]
```diff
@@ -4,6 +4,10 @@ package:
 source:
   url: https://files.pythonhosted.org/packages/78/fd/0148f0e54f0c6f48a141409df65d74a5f1dae2e139f23d50a43c58c16098/pyerfa-2.0.0.1.tar.gz
   sha256: 2fd4637ffe2c1e6ede7482c13f583ba7c73119d78bef90175448ce506a0ede30
+  patches:
+    - patches/0001-Fix-compilation-with-Wincompatible-function-pointer-.patch
+build:
+  script: python3.10 erfa_generator.py
```
Comment (ryanking13) [inline packages/pyerfa/meta.yaml:10]: What does this command do?

**Review by ryanking13 — APPROVED** [review]
Thanks, @hoodmane! It seems like every Emscripten update is a challenge.

File: packages/pyerfa/meta.yaml (hunk 1 above)
Comment (hoodmane) [reply packages/pyerfa/meta.yaml]: See added comments.

**Comment by hoodmane** [comment]
> It seems like every Emscripten update is a challenge.

Yeah it's always exciting. At least this was a relatively easy one.

**Comment by rth** [comment]
Regarding that CPython version / Emscripten version policy, does it mean that we only enforce it from next version and not this one? Or do we aim to update CPython for the next release?

**Comment by hoodmane** [comment]
I think we're starting from next release. Is that what we've decided @ryanking13?

I think it would be a good idea to work on some emscripten related automation. It would be nice to automatically try out each emscripten-releases commit...

**Pull request merged**